app/src/androidTest/java/com/duckduckgo/app/statistics/VariantManagerTest.kt

```diff
@@ -23,36 +23,35 @@ import org.junit.Test
 class VariantManagerTest {
```

Comment:
Not your changes, but it would be to good to format this class as part of this PR since we're in here anyway 🙏